tests/product/test_server_upgrade.py

```diff
@@ -132,12 +132,33 @@ def test_upgrade_works_with_symlink(self):
         self.upgrade_and_assert_success(symlink)
 
     def test_configuration_preserved_on_upgrade(self):
+        book_content = 'Call me Ishmael ... FINIS'
+        book_path = '/etc/presto/moby_dick_abridged'
         self.run_prestoadmin('configuration deploy')
+        big_files = {}
         for container in self.cluster.all_hosts():
             self.real_installer.assert_installed(self, container)
             self.assert_has_default_config(container)
             self.assert_has_default_catalog(container)
 
+            big_file = self.cluster.exec_cmd_on_host(
+                container,
+                "find /usr -size +2M -ls | "
```

Comment:
There are a fair number of files that match. This runs on a cluster that has presto installed, so there almost 2 dozen copies of guava installed at the very least :-b